llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -483,6 +489,66 @@ bool RISCVInstructionSelector::selectConstant(MachineInstr &MI,
   return true;
 }
 
+bool RISCVInstructionSelector::selectGlobalValue(
+    MachineInstr &MI, MachineIRBuilder &MIB, MachineRegisterInfo &MRI) const {
+  assert(MI.getOpcode() == TargetOpcode::G_GLOBAL_VALUE &&
+         "Expected G_GLOBAL_VALUE");
+  auto GV = MI.getOperand(1).getGlobal();
+  if (GV->isThreadLocal()) {
+    // TODO: implement this case.
+    return false;
+  }
+
+  switch (TM.getCodeModel()) {
+  default:
+    report_fatal_error("Unsupported code model for lowering");
```

Comment:
```
reportGISelFailure(const_cast<MachineFunction &>(MF), *TPC, *MORE,
                       "riskv-selector", "unknown code model",
                       MI);
```
turns the error into a remark.

Reply:
64daec66e4d03d0a002d3973bc25dd7cedf312e8